frontend/src/app/modules/posts/components/sort-comments/sort-comments.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-sort-comments',
+  templateUrl: './sort-comments.component.html'
+})
+export class SortCommentsComponent {
+  @Output() sort: EventEmitter<{ sortBy: string; sortOrder: string }> = new EventEmitter<{ sortBy: string; sortOrder: string }>();
+  sortOrderTitle: string = "Descending ⮟";
+  sortByTitle: string = "date";
+  sortBy: string = "createdAt";
+  sortOrder: string = "DESC";
```

Comment:
Co sądzisz o daniu tutaj typu, który będzie nam to jeszcze zabezpieczał?
np enum stworzyć albo dać typ w stylu: `'DESC' | 'ASC'`